libraries/node-core-library/src/Terminal/StringBufferTerminalProvider.ts

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import { ITerminalProvider, Severity } from './ITerminalProvider';
+
+/**
+ * @beta
+ */
```

Comment:
Maybe wait to export this until we have another project that needs it.  The use case is a little unclear to me.